### PR TITLE
ADD: Toggle for RBF

### PR DESCRIPTION
**Pull request by marcosrdz**
Simulator Screen Shot - iPhone X - 2019-12-10 at 23 12 58

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
This was a triumph. I'm making a note here: HUGE SUCCESS.

 [android] https://appetize.io/app/085zy2wm55vb3c7epb3edwm01m

**Comment by GladosBlueWallet** [comment]
This was a triumph. I'm making a note here: HUGE SUCCESS.

 [android] https://appetize.io/app/6q1pff9q4vu0uvywaebd2zp9hm

**Comment by GladosBlueWallet** [comment]
This was a triumph. I'm making a note here: HUGE SUCCESS.

 [android] https://appetize.io/app/nh1p9vjaaj6844k5f2zducbeym

File: class/hd-segwit-bech32-wallet.js [hunk 1]
```diff
@@ -727,7 +728,11 @@ export class HDSegwitBech32Wallet extends AbstractHDWallet {
    */
   createTransaction(utxos, targets, feeRate, changeAddress, sequence, skipSigning = false) {
     if (!changeAddress) throw new Error('No change address provided');
-    sequence = sequence || HDSegwitBech32Wallet.defaultRBFSequence;
+    if (sequence === false) {
```
Comment (Overtorment) [inline class/hd-segwit-bech32-wallet.js]: I have just added `HDSegwitBech32Wallet.finalRBFSequence` constant. pass it from outside  (to `createTransaction()`)

**Comment by GladosBlueWallet** [comment]
This was a triumph. I'm making a note here: HUGE SUCCESS.

 [android] https://appetize.io/app/9cugw6kdqkpb8dbb5wwrc9mgpw

File: class/hd-segwit-bech32-wallet.js (hunk 1 above)
Comment (marcosrdz) [reply class/hd-segwit-bech32-wallet.js]: Done

**Pull request merged**